docs/01-app/03-building-your-application/07-configuring/16-debugging.mdx

```diff
@@ -151,15 +151,7 @@ Debugging server-side code works similarly to client-side debugging. When search
 
 When you encounter an error, inspecting the source code can help trace the root cause of errors.
 
-Next.js will display a Node.js logo like a green button on the dev overlay. By clicking that button, the DevTools URL is copied to your clipboard. You can open a new browser tab with that URL to inspect the Next.js server process.
-
-<Image
-  alt="Copy DevTools url example"
-  srcLight="/docs/dark/copy-devtool-url-example.png"
-  srcDark="/docs/dark/copy-devtool-url-example.png"
-  width="1600"
-  height="594"
-/>
+Next.js will display a Node.js logo underneath the Next.js version indicator on the error overlay. By clicking that button, the DevTools URL is copied to your clipboard. You can open a new browser tab with that URL to inspect the Next.js server process.
```

Comment:
Branding police here: It's the Node.js **icon**. The logo is a stylized version of "Node.js" where the "o" is replaced by the Node.js icon (https://nodejs.org/static/documents/foundation-visual-guidelines.pdf) 😄